cfy_manager/components/validations.py

```diff
@@ -555,8 +559,8 @@ def _validate_postgres_inputs():
                                   'postgres_password must be set')
 
     if is_installed(MANAGER_SERVICE) and not is_installed(DATABASE_SERVICE):
-        postgres_host = config[POSTGRESQL_CLIENT]['host'].split(':')[0]
-        if postgres_host in ('localhost', '127.0.0.1') and \
+        postgres_host = config[POSTGRESQL_CLIENT]['host'].rsplit(':', 1)[0]
```

Comment:
Are there no utility functions for doing this in any of the deps we already bring in? I thought we had ipaddr or something which should be able to tokenise addresses/ports?

Reply:
Oh, you're right, we do use ipaddress in this repo, cool.